### ISSUE TITLE
Standalone C++ with Admob

### ISSUE THREAD

**Issue by wini3d** [sdkbox/sdkbox-sample-admob#4]
Hi, I came across this on the website where it says _AdMob with C++_, so can I use this library without using cocos2d-x?

**Comment by yinjimmy** [comment]
yes.
this library used with unity.

**Comment by wini3d** [comment]
Thanks @yinjimmy 

Is there a documentation on how to integrate it with just Native C++ code? I only found this guide here: https://docs.sdkbox.com/en/plugins/admob/v3-cpp/

but it says _For the C++ version of cocos2d-x v3.x_

Also, do you need to use JNI for this?

**Comment by yinjimmy** [comment]
iOS: c++ <-- objc
Android: c++ - JNI - java


-----

The doc is also can work with purge c++ project.
Need JNI when you use it for Android platform.

**Comment by wini3d** [comment]
Thanks. Are there any examples for Android C++ JNI version?

**Comment by yinjimmy** [comment]
do you have Android Project ?
SDKBox's AdMob Plugin wraps the c++ ----> jni ---> java detail.
whats your need is PluginAdMob c++ api.

**Comment by wini3d** [comment]
> do you have Android Project ?
Yes, I am using a simple native c++ example and building via ndk-build and apache ant.

> whats your need is PluginAdMob c++ api.
Thanks @yinjimmy  Where can I get that?

> yes, it works without cocos2d-x.
perhaps you can provide a sample repo and/or build setup for non cocos-2dx users?

**Comment by yinjimmy** [comment]
> via ndk-build and apache ant.

It's better move to Android Studio project.
SDKBox do not provide `google play services` library anymore.

**Comment by wini3d** [comment]
>It's better move to Android Studio project.
> SDKBox do not provide google play services library anymore.

Ok thanks for the info. Is there an Android Studio example project that is not cocos2d-x to try out?

**Comment by yinjimmy** [comment]
I make a example https://github.com/sdkbox/sdkbox-sample-purecpp

**Comment by wini3d** [comment]
@yinjimmy This is awesome! thank you so much! :heart: 

I will try it and let you know if I encounter some issues.
